metrics/cli/test/deploy/deployer_test.dart

```diff
@@ -469,6 +477,33 @@ void main() {
       },
     );
 
+    test(
+      ".deploy() adds Firebase to the GCloud project with the created project id",
```

Comment:
Should we test that this method adds the Firebase to the GCloud project before creating the Firebase Web app?